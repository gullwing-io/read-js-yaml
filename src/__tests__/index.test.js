import readJsYaml from "../../index.js";
  
  describe("Loading YAML", () => {
    describe("readJsYaml", () => {
      test("should return locale bookmarks as JSON", async () => {
        expect(
          await readJsYaml("./src/__tests__/__stubs__/bookmarks.yaml")
        ).toMatchSnapshot();
      });
    });
  });
  