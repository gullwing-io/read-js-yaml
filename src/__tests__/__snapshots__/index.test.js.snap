// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading YAML readJsYaml should return locale bookmarks as JSON 1`] = `
{
  "body": {
    "description": "These are sample bookmarks to teach you how Bookworms works",
    "folders": [
      {
        "description": "This is to describe the folder structure",
        "folders": [
          {
            "bookmarks": [
              {
                "description": "this is used to describe the bookmark",
                "href": "https://www.mywebsite.com",
                "label": "sample url 1",
              },
            ],
            "description": "This is to describe the sub folder structure",
            "label": "sub folder 1",
          },
        ],
        "label": "folder 1",
      },
      {
        "folders": [
          {
            "bookmarks": [
              {
                "description": "this is used to describe the bookmark",
                "href": "https://www.mywebsite.com",
                "label": "sample url 2",
              },
              {
                "description": "this is used to describe the bookmark",
                "href": "https://www.mywebsite.com",
                "label": "sample url 3",
              },
            ],
            "label": "sub folder 2",
          },
          {
            "bookmarks": [
              {
                "description": "this is used to describe the bookmark",
                "href": "https://www.mywebsite.com",
                "label": "sample url 4",
              },
              {
                "description": "this is used to describe the bookmark",
                "href": "https://www.mywebsite.com",
                "label": "sample url 5",
              },
            ],
            "label": "sub folder 3",
          },
        ],
        "label": "folder 2",
      },
    ],
    "label": "Bookworms",
  },
  "type": "local",
}
`;
